### PR TITLE
add feed command which will launch a note from source

### PR DESCRIPTION
**Pull request by awfulcarpet**
feeds to an alliance partner at amp/speaker over the stage

### CODE REVIEW

File: src/main/java/frc/robot/commands/FeedCommand.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        shooter.startMotors(speed);
+        shooter.setNoteReadyToFire(true);
+    }
+
+    // Returns true when the command should end.
+    @Override
+    public boolean isFinished() {
+        return shooter.isAngled();
+    }
+}
```
Comment (centiwati) [inline src/main/java/frc/robot/commands/FeedCommand.java:35]: Pretty sure we don't need this and can just use PrepCommand

File: src/main/java/frc/robot/subsystems/Shooter.java [hunk 2]
```diff
@@ -51,6 +51,11 @@ public TalonFX getPivotMotor() {
         return pivotMotor;
     }
 
+    public boolean isAngled() {
```
Comment (centiwati) [inline src/main/java/frc/robot/subsystems/Shooter.java:54]: Why do we have this?

File: src/main/java/frc/robot/commands/FeedCommand.java (hunk 1 above)
Comment (awfulcarpet) [reply src/main/java/frc/robot/commands/FeedCommand.java:35]: Nathaniel wanted a feed command that only cared about launching a note towards the amp/speaker without needing to wait for the motors to rev up fast enough, but with the correct angle

File: src/main/java/frc/robot/subsystems/Shooter.java (hunk 2 above)
Comment (awfulcarpet) [reply src/main/java/frc/robot/subsystems/Shooter.java:54]: to return whether just the position of the arm is correct

**Review by centiwati — APPROVED** [review]

**Comment by awfulcarpet** [comment]
unneeded as speakershoot was sufficient

**Pull request closed without merging**